packages/utils/lib/helpers/helpers.ts

```diff
@@ -609,3 +610,64 @@ export function getInviteParticipantStepAnswers(): PatchPaperworkParameters['ans
     ],
   };
 }
+
+export async function getPatientConditionPhotosStepAnswers({
+  intakeZambdaUrl,
+  authToken,
+  projectId,
+  appointmentId,
+  filePromise,
```

Comment:
```suggestion
  fileUploadPromise,
```

maybe if it's just awaited here then we can just pass the uploaded file url instead of promise as parameter?

Reply:
and if it's not provided (optional) - do not fill patient condition photo for this step, maybe we'll use this in some other scenario for example to check that there are no patietn condition photos for subsequent appointment for the same patient